packages/bot-skeleton/src/utils/error-config.js

```diff
@@ -0,0 +1,23 @@
+import { localize } from '@deriv/translations';
+
+const generateErrorMessage = (blockType, missing_space = 'workspace') => {
+    return {
+        missing: localize(`The ${blockType} block is mandatory and cannot be deleted/disabled.`),
+        misplaced: localize(`The ${blockType} block is misplaced from ${missing_space}.`),
+        disabled: localize(`The ${blockType} block is mandatory and cannot be deleted/disabled.`),
+        default: localize(`The ${blockType} block is mandatory and cannot be deleted/disabled.`),
+    };
```

Comment:
Can we do this instead? @rupato-deriv let me know if this doesn't work.
```suggestion
    return {
        missing: localize('The {{block_type}} block is mandatory and cannot be deleted/disabled.', { block_type: blockType }),
        misplaced: localize('The {{block_type}} block is misplaced from {{missing_space}}.', { block_type: blockType, missing_space: space }),
        disabled: localize('The {{block_type}} block is mandatory and cannot be deleted/disabled.', { block_type: blockType }),
        default: localize('The {{block_type}} block is mandatory and cannot be deleted/disabled.', { block_type: blockType }),
    };
```

Reply:
done ✅